components/style/themes/default.less

```diff
@@ -383,6 +383,9 @@
 @menu-item-active-bg: @item-active-bg;
 @menu-item-active-border-width: 3px;
 @menu-item-group-title-color: @text-color-secondary;
+@menu-item-vertical-margin: 4px;
+@menu-item-font-size: @font-size-base;
+@menu-item-last-margin: 8px;
```

Comment:
`menu-item-last-margin` is little strange. How about `menu-item-boundary-margin`?

Reply:
changed menu-item-last-margin to  menu-item-boundary-margin